tests/WE2E/machine_suites/comprehensive.derecho

```diff
@@ -0,0 +1,55 @@
+community
```

Comment:
This can be the a symbolic link to comprehensive.orion, just like comprehensive.gaea (as of PR #871). Unlike Cheyenne, Derecho's compute nodes have access to the internet. I tested the 3 missing tests from comprehensive.cheyenne (`2020_CAD`, `get_from_AWS_ics_GEFS_lbcs_GEFS_fmt_grib2_2022040400_ensemble_2mems`, and `get_from_NOMADS_ics_FV3GFS_lbcs_FV3GFS`) and they all passed.